redfish/powerdistribution.go

```diff
@@ -139,7 +139,7 @@ type PowerDistribution struct {
 	powerSupplies string
 	// Deprecated: (v1.3) The redundancy information for the devices in a redundancy group.
 	// This property has been deprecated in favor of the PowerSupplyRedundancy property in the Chassis resource.
-	PowerSupplyRedundancy RedundantGroup
+	PowerSupplyRedundancy []RedundantGroup
```

Comment:
Did you mean to include this here?

Reply:
Ah yes, sorry, I forgot to make a separate commit with this fix
(Specs: PowerSupplyRedundancy [ { } ] | array (object))